apps/meteor/client/views/admin/apps/components/RadioDropDown/RadioDropDown.tsx

```diff
@@ -9,7 +9,7 @@ import DropDownListWrapper from '../DropDownListWrapper';
 import RadioButtonList from '../RadioButtonList';
 import RadioDropDownAnchor from './RadioDownAnchor';
 
-const RadioDropDown: FC<RadioDropDownProps & Partial<ComponentProps<typeof Select>>> = ({ group, onSelected, ...props }) => {
+const RadioDropDown = ({ group, onSelected, ...props }: RadioDropDownProps & Partial<ComponentProps<typeof Select>>): ReactElement => {
```

Comment:
I would suggest not to type with all Select props, but to define specifically the ones that are being used. Or is there a reason for forwarding all "rest" props? It increases the chance of unwanted/unforeseen props being passed.

Reply:
The idea was for RadioDropDown to be a direct abstraction of Select that's why I used all props of Select so that both components can be used interchangeably.